index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  configs: {
+    tslint: require('./lib/config/all.js'),
```

Comment:
Yah, I believe we should go with [tslint-config-shopify](https://www.npmjs.com/search?q=tslint+config)